README.md

```diff
@@ -1,11 +1,21 @@
-AdvancedTabOverlay is a plugin for Spigot that has not been documented by its author yet.
+[Dev Builds]: https://ci.codecrafter47.de/job/AdvancedTabOverlay/
+[ProtocolLib]: https://www.spigotmc.org/resources/1997/
+[Wiki]: https://github.com/CodeCrafter47/AdvancedTabOverlay/wiki
 
-File locations
---------------
-- `src/site/markdown`: Markdown files that will be included in the generated site
-- `src/main/java`: Main plugin source files
-- `src/test/java`: Test source files
+# AdvancedTabOverlay
+AdvancedTabOverlay is a tab list plugin for Spigot, which provides highly customisable features including but not limited to:
+- Fake Players (Custom slots)
```

Comment:
Preferably this would just say _Custom slots_. I don't refer to them as _Fake Players_.